ms/MSOper/MSMetaData.h

```diff
@@ -85,12 +85,13 @@ class MSMetaData {
         // the subscan and that spwID
         map<uInt, Quantity> meanInterval;
         Quantity meanExposureTime;
-        uInt nrows;
         std::set<uInt> spws;
         // number of rows for each spectral window
         map<uInt, uInt> spwNRows;
         std::set<Int> stateIDs;
-        std::map<Double, TimeStampProperties> timeProps;
+        map<Double, TimeStampProperties> timeProps;
+        // number of cross-correlation rows.
+        uInt xcRows;
```

Comment:
this should maybe be placed next to acRows as it is related
as you are breaking binary compat anyway be removing nrows you don't need to add at the bottom.